### DISCUSSION TITLE
VSCode Extension

### DISCUSSION THREAD

**Discussion by mylastresort** [ahrm/sioyek#349] [Ideas]
Greetings !

Man I just love your application, could you think about making a vscode extension release, I would really like to have it next to this one: [coolest whiteboard ever](https://marketplace.visualstudio.com/items?itemName=pomdtr.excalidraw-editor)

**Comment by ryanwwest** [comment]
I think a VSCode Extension would be neat as well, and really a Chrome/Firefox extension too. Since VSCode is electron-based, its extensions are written in Javascript or Typescript so I think the major hurdle here would be getting sioyek to work on that first. After that, I think a major chunk of the work would be done for both browsers and VSCode (and also literally for just serving sioyek from the browser, not necessarily needing an extension).
